compiler/src/dotty/tools/dotc/typer/RefChecks.scala

```diff
@@ -767,7 +767,7 @@ object RefChecks {
           for (mbrd <- self.member(name).alternatives) {
             val mbr = mbrd.symbol
             val mbrType = mbr.info.asSeenFrom(self, mbr.owner)
-            if (!mbrType.overrides(mbrd.info, matchLoosely = true))
+            if (!mbrType.overrides(mbrd.info, false, matchLoosely = true))
```

Comment:
```suggestion
            if (!mbrType.overrides(mbrd.info, relaxedCheck = false, matchLoosely = true))
```

Reply:
Agreed. Every true/false argument should be by name.